llm/codellama/README.md

```diff
@@ -0,0 +1,185 @@
+# Serving Code Llama
+
+[Code Llama](https://github.com/facebookresearch/codellama) is a code-specialized version of Llama 2 that was created by further training Llama 2 on its code-specific datasets, sampling more data from that same dataset for longer. On Jan 29th, 2024, Meta releases the Code Llama 70B, the largest and best-performing model in the Code Llama family.
+
+## References
+* [Llama-2 Example](../../llm/llama-2/)
+* [Code Llama release](https://ai.meta.com/blog/code-llama-large-language-model-coding/)
+* [Code Llama paper](https://arxiv.org/abs/2308.12950)
+
+
+## Why use SkyPilot/SkyServe to deploy over commercial hosted solutions?
+
+* Get the best GPU availability by utilizing multiple resources pools across multiple regions and clouds.
+* Enable scaling up to multiple replicas across different locations and accelerators with a single endpoint exposed.
+* Everything stays in your cloud account (your VMs & buckets)
+* No one else sees your chat history
+* Pay absolute minimum — no managed solution markups
+
+
+## Running your own Code Llama with SkyPilot
+
+You can now host your own Code Llama with SkyPilot using 1-click.
```

Comment:
Should we add a quick link to the installation instructions for people who may be new to SkyPilot?:
```suggestion
After [installing SkyPilot](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html), run your own Code Llama on vLLM with SkyPilot in 1-click:
```